specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/StaticSites.json

```diff
@@ -999,6 +999,61 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/builds/{environmentName}/purge": {
+      "post": {
+        "tags": [
+          "StaticSites"
+        ],
+        "summary": "Purges a static site build.",
+        "description": "Description for Purges a static site build.",
+        "operationId": "StaticSites_PurgeStaticSiteBuild",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the static site to purge.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "The stage site identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=11&length=3)

ARM requires all APIs to be either always sync or always async. Since you are supporting async, you don't need sync status codes.